workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/hooks/useCompanyOptions.ts

```diff
@@ -52,12 +52,34 @@ export default function useCompanyOptions(
 			`/o/${LiferayAPIs.OBJECT}/partnerlevels/by-external-reference-code/${account.r_prtLvlToAcc_c_partnerLevelERC}`
 	);
 
-	const countryPicklist =
-		account &&
-		countryOptions &&
-		countryOptions.find(
-			(options) => options.value === account.partnerCountry
-		);
+	const formatPartnerCountries = useCallback(
+		(
+			company?: AccountEntry,
+			countriesList?: React.OptionHTMLAttributes<HTMLOptionElement>[]
+		) => {
+			if (!isObjectEmpty(company) && !!countriesList?.length) {
+				const splitedCountries = company?.partnerCountry?.split(', ');
```

Comment:
can we rename this variable to `splitCountries`

Reply:
Done